test/lib/gas/GasTestBase.t.sol

```diff
@@ -40,6 +40,11 @@ abstract contract GasTestBaseStored {
     console.log("Description: %s", description);
   }
 
+  function printDescription(string memory postfix) public virtual {
+    description = string.concat(description, postfix);
```

Comment:
I'm surprised that this changes the description. If this is called multiple times, then the description grows each time?

Perhaps it would be more predictable, if the postfix was only printed this once? Eg by having this function do the actual printing and `printDescription()` forward to this with an empty postfix?

Reply:
agree :)